### PR TITLE
Optimize cardboard cutouts saving 1.5s+ on init times

### LINKED ISSUE #32: Cardboard cutout generation is way too expensive
`cardboard_cutout/Initialize`, despite being called twice, is 1.47 seconds, from `/proc/generate_cardboard_cutouts`, which generates every single cutout using flat icons. These should be changed into normal appearances, or at the very least not be done all at once.

### PR DESCRIPTION
**Pull request by Mothblocks**
New regression in init times. Closes https://github.com/tgstation/dev-cycles-initiative/issues/32. CC @Fikou 

- Instead of creating a human and icon for *every* cardboard cutout when initialized, only creates the one we're actually using. When you're about to use a crayon, creates all of them.
- Instead of using getFlatIcon, uses appearances directly.

### CODE REVIEW

File: code/game/objects/items/cardboard_cutouts.dm [hunk 1]
```diff
@@ -140,22 +155,17 @@ GLOBAL_LIST(cardboard_cutouts)
 		return FALSE
 	return TRUE
 
+/obj/item/cardboard_cutout/setDir(newdir)
```
Comment (Mothblocks) [inline code/game/objects/items/cardboard_cutouts.dm:158]: Needed now that we're not using a flat icon that only has one direction

**Comment by Fikou** [comment]
you kept generated dirs even though the functionality was removed

**Comment by Mothblocks** [comment]
i wish you put that as a comment on the parameters instead because it took me an entire day to figure out what on earth you were talking about

File: code/__HELPERS/dynamic_human_icon_gen.dm [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ///Global list of all dynamically generated icons, for caching, so we don't have to generate multiple times.
-GLOBAL_LIST_EMPTY(dynamic_human_icons)
+GLOBAL_LIST_EMPTY(dynamic_human_appearances)
 
-///This proc takes an amount of arguments and creates a human dummy using the paths, for visuals you want!
-/proc/get_dynamic_human_icon(outfit_path, species_path = /datum/species/human, mob_spawn_path, r_hand, l_hand, bloody_slots = NONE, list/generated_dirs = GLOB.cardinals, animated = TRUE)
+/// Creates a human with the given parameters and returns an appearance of it
+/proc/get_dynamic_human_appearance(outfit_path, species_path = /datum/species/human, mob_spawn_path, r_hand, l_hand, bloody_slots = NONE, list/generated_dirs = GLOB.cardinals, animated = TRUE)
```
Comment (Mothblocks) [inline code/__HELPERS/dynamic_human_icon_gen.dm:5]: they are talking about `generated_dirs` here

File: code/game/objects/items/extinguisher.dm [hunk 3]
```diff
@@ -255,9 +255,9 @@
 /obj/item/extinguisher/proc/manage_chair_speed(datum/move_loop/move/source)
 	SIGNAL_HANDLER
 	switch(source.lifetime)
-		if(5 to 4)
+		if(4 to 5)
```
Comment (Mothblocks) [inline code/game/objects/items/extinguisher.dm:258]: Accidentally committed this but also this is already in maste rso whatever

**Pull request merged**